service/handlers_test.go

```diff
@@ -91,7 +91,7 @@ func (m mockWorkflowSvc) List(ctx context.Context) ([]string, error) {
 }
 
 func (m mockWorkflowSvc) Submit(ctx context.Context, from string, parameters map[string]string) (string, error) {
-	return "success", nil
+	return "wf-123456", nil
```

Comment:
I changed this to be closer so something that we might expect. I actually thought `success` was coming from somewhere else because I was expecting some type of id when refactoring.